bsc.verilog/derived_bits/derived_bits.exp

```diff
@@ -42,3 +42,12 @@ compare_verilog c0Test.v
 compare_verilog c1Test.v
 
 compare_verilog origTest.v
+
+compile_verilog_pass DerivedEnums.bsv
```

Comment:
This looks like a duplicate of the line below (and should be deleted?)

Reply:
Thanks for catching these, fixed.